### PR TITLE
add missing `v` to tag

### PR DESCRIPTION
**Pull request by winmillwill**
so that copy paste works

### Prerequisites

Put an `x` into the box(es) that apply:

- [x] This pull request fixes a bug.
- [ ] This pull request adds a feature.
- [ ] This pull request enhances existing functionality.
- [ ] This pull request introduces breaking change.

For more information, see the [Contributing Guide](https://github.com/terraform-docs/terraform-docs/tree/master/CONTRIBUTING.md).

### Description

Adds missing `v` to the tag for latest version to the README so that copy-pasting will work.

### Issues Resolved

Copy pasting of recommended install path was broken.

### Checklist

Put an `x` into all boxes that apply:

- [x] I have read the [Contributing Guidelines](https://github.com/terraform-docs/terraform-docs/tree/master/CONTRIBUTING.md).

#### Tests

- [ ] I have added tests to cover my changes.
- [ ] All tests pass when I run `make test`.

#### Documentation

- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.

#### Code Style

- [x] My code follows the code style of this project.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ Visit [./docs](./docs/) for all documentation.
 The latest version can be installed using `go get`:
 
 ```bash
-GO111MODULE="on" go get github.com/terraform-docs/terraform-docs@0.10.1
+GO111MODULE="on" go get github.com/terraform-docs/terraform-docs@v0.10.1
```
Comment (khos2ow) [inline README.md:43]: Very nice catch!

**Review by khos2ow — COMMENTED** [review]
I actually went through the whole file and found out we're missing `v` in line 63 too, and the version number in line 85 is also outdated. Can you fix them too?

**Comment by winmillwill** [comment]
@khos2ow done

File: README.md [hunk 2]
```diff
@@ -82,7 +82,7 @@ choco install terraform-docs
 Alternatively you also can run `terraform-docs` as a container:
 
 ```bash
-docker run quay.io/terraform-docs/terraform-docs:0.10.0
+docker run quay.io/terraform-docs/terraform-docs:v0.10.1
```
Comment (khos2ow) [inline README.md:85]: This one shouldn't have the `v`

**Review by khos2ow — CHANGES_REQUESTED** [review]

**Review by khos2ow — APPROVED** [review]
LGTM!

**Pull request merged**